sql_generators/experiment_monitoring/templates/experiment_events_live_v1/materialized_view.sql

```diff
@@ -1,7 +1,6 @@
 -- Generated via ./bqetl generate experiment_monitoring
-CREATE MATERIALIZED VIEW
-IF
-  NOT EXISTS {{ dataset }}_derived.{{ destination_table }}
+CREATE OR REPLACE MATERIALIZED VIEW
```

Comment:
Some code assumes all materialized views start with `CREATE MATERIALIZED VIEW` ([example 1](https://github.com/mozilla/bigquery-etl/blob/08bd44a73ca223361ef9f3ccef1c2c6a07fb99d6/bigquery_etl/cli/stage.py#L175), [example 2](https://github.com/mozilla/bigquery-etl/blob/08bd44a73ca223361ef9f3ccef1c2c6a07fb99d6/bigquery_etl/dryrun.py#L218)), so I wouldn't recommend merging this as is.

And as Ben pointed out in another comment, we're intentionally not automatically redeploying materialized views (with the obvious drawback that redeploying them requires manual intervention).